databuilder/extractor/neo4j_search_data_extractor.py

```diff
@@ -24,18 +24,22 @@ class Neo4jSearchDataExtractor(Extractor):
         {publish_tag_filter}
         OPTIONAL MATCH (table)-[:DESCRIPTION]->(table_description:Description)
         OPTIONAL MATCH (schema)-[:DESCRIPTION]->(schema_description:Description)
+        OPTIONAL MATCH (table)-[:DESCRIPTION]->(prog_descs:Programmatic_Description)
```

Comment:
Could you perform `WITH ... COLLECT(prog_descs.description)` right after this? Neo4j is performance degrades if it's not aggregated early.

Reply:
ok interesting thanks, will modify.